compiler/cpp/src/generate/t_java_generator.cc

```diff
@@ -2579,7 +2584,15 @@ void t_java_generator::generate_service_future_interface(t_service* tservice) {
   vector<t_function*> functions = tservice->get_functions();
   vector<t_function*>::iterator f_iter;
   for (f_iter = functions.begin(); f_iter != functions.end(); ++f_iter) {
-    indent(f_service_) << function_signature_future(*f_iter) << ";" << endl << endl;
+    t_function* t_f = *f_iter;
+    int mask;
```

Comment:
Yes, it's safe. If you want to be style-consistent, then leave "int i" on a separate line, then initialize it in the for-loop.